docs/scenarios_authoring/Spel.md

```diff
@@ -276,27 +276,28 @@ More usage examples can be found in [this section](#conversions-between-datetime
 
 #### Explicit conversions
 
-Explicit conversions are available in utility classes and build-in java conversion mechanisms:
-
-| Expression                                                      | Result                     | Type            |
-|-----------------------------------------------------------------|----------------------------|-----------------|
-| `#NUMERIC.toNumber('42')`                                       | 42                         | Number          |
-| `#NUMERIC.toNumber('42').toString()`                            | '42'                       | String          |
-| `#DATE_FORMAT.parseOffsetDateTime('2018-10-23T12:12:13+00:00')` | 1540296720000              | OffsetDateTime  |
-| `#DATE_FORMAT.parseLocalDateTime('2018-10-23T12:12:13')`        | 2018-10-23T12:12:13+00:00  | LocalDateTime   |
-
-### Casting
-
-When a type cannot be determined by parser, the type is presented as `Unknown`. When we know what the type will be on
-runtime, we can cast a given type, and then we can operate on the cast type.
-
-E.g. having a variable `obj` of a type: `List[Unknown]` and we know the elements are strings then we can cast elements
-to String: `#obj.![#this.castToOrNull('String')]`.
-
-Available methods:
-- `canCastTo` - checks if a type can be cast to a given class.
-- `castTo` - casts a type to a given class or throws exception if type cannot be cast.
-- `castToOrNull` - casts a type to a given class or return null if type cannot be cast.
+Explicit conversions are available in utility classes, build-in java conversion mechanisms and via extension methods:
```

Comment:
As a newcomer, I would have many question marks after this introduction and table. They describe that they have three options to convert something to something, they show some examples, I can't see full set of conversion, what approach should I use in my case? Take a look at how Flink deals with this: https://nightlies.apache.org/flink/flink-docs-master/docs/dev/table/types/#casting